{"results":[{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"LessThan\n","instructorExeTime":23,"studentStdOut":"LessThan\n","studentExeTime":3,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":139,"studentErrOut":"Seg fault","instructorStdOut":"Inbetween boundaries\n","instructorExeTime":3,"studentStdOut":"","studentExeTime":3,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":139,"studentErrOut":"Seg fault","instructorStdOut":"Inbetween boundaries\n","instructorExeTime":3,"studentStdOut":"","studentExeTime":2,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"6\n","instructorExeTime":78,"studentStdOut":"6\n","studentExeTime":71,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":1,"studentErrOut":"Exception in thread \"main\" java.lang.NumberFormatException: For input string: \"4\" under radix 2\n\tat java.base\/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)\n\tat java.base\/java.lang.Long.parseLong(Long.java:711)\n\tat Main.main(Main.java:4)\n","instructorStdOut":"Invalid value\n","instructorExeTime":46,"studentStdOut":"","studentExeTime":44,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"Stream closed","instructorStdOut":"Finished Loop\n","instructorExeTime":73,"studentStdOut":"","studentExeTime":5007,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"","instructorExeTime":79,"studentStdOut":"","studentExeTime":102,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"truefalsetruefalsetruefalsetruefalsetruefalsefalsefalsetruefalsetruefalsetruefalsetruefalse\n","instructorExeTime":79,"studentStdOut":"truefalsetruefalsetruefalsetruefalsetruefalsefalsefalsetruefalsetruefalsetruefalsetruefalse\n","studentExeTime":80,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"Exception in thread \"Thread-1\" Exception in thread \"Thread-0\" java.lang.AssertionError: RaceCondition occurred\n\tat Number.run(Module9.java:36)\njava.lang.AssertionError: RaceCondition occurred\n\tat Number.run(Module9.java:36)\n","instructorStdOut":"","instructorExeTime":44,"studentStdOut":"","studentExeTime":46,"instructorErrOut":""},{"instructorExitCode":0,"studentExitCode":0,"studentErrOut":"","instructorStdOut":"0\n1\nThread with Priority 0 executed\nThread with Priority 1 executed\n2\n3\nThread with Priority 2 executed\n4\nThread with Priority 3 executed\n5\nThread with Priority 4 executed\n6\nThread with Priority 5 executed\n7\nThread with Priority 6 executed\nThread with Priority 7 executed\n8\n9\n10\nThread with Priority 9 executed\nThread with Priority 8 executed\n0\n1\n2\n3\n","instructorExeTime":77,"studentStdOut":"0\n0\n1\n0\n1\n2\nThread with Priority 1 executed\nThread with Priority 0 executed\n0\n1\n2\n3\nThread with Priority 2 executed\n0\n1\n2\n3\nThread with Priority 3 executed\n","studentExeTime":81,"instructorErrOut":""}]}